### PR TITLE
Create MainMenu class.

### PR DESCRIPTION
**Pull request by lmunozm1702**
### The console app, at the start, should:
- Present the user with a list of options to perform.
- Let users choose an option.
- If needed, ask for parameters for the option.

### The following options should be available:
- List all books
- List all music albums
- List all movies
- List of games
- List all genres (e.g 'Comedy', 'Thriller')
- List all labels (e.g. 'Gift', 'New')
- List all authors (e.g. 'Stephen King')
- List all sources (e.g. 'From a friend', 'Online shop')
- Add a book
- Add a music album
- Add a movie
- Add a game

### CODE REVIEW

File: main.rb [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  def exit_app
+    puts 'Thank you for using the app!'
+    exit(0)
+  end
+
+  def main_menu
+    show_options
+    user_input = gets.chomp.to_s.upcase
+    exit_app if user_input == 'X'
+    send(@options[user_input])
```
Comment (HevarKas) [inline main.rb]: Please also add that when a user enters an invalid character, it should display an error message.

**Review by HevarKas — CHANGES_REQUESTED** [review]
check comment

File: main.rb (hunk 1 above)
Comment (lmunozm1702) [reply main.rb]: Hi @whiteWolfx99 it's done!

Comment (HevarKas) [reply main.rb]: ok well done

**Review by HevarKas — APPROVED** [review]
well done you merge it now

**Comment by lmunozm1702** [comment]
Merge 
Done!

**Pull request merged**